Cake.BenchmarkDotNet/Comparer.cs

```diff
@@ -1,21 +1,29 @@
-﻿using System;
+﻿using Cake.BenchmarkDotNet.Dto;
+using Newtonsoft.Json;
+using Perfolizer.Mathematics.SignificanceTesting;
+using Perfolizer.Mathematics.Thresholds;
+using System;
 using System.Collections.Generic;

```

Comment:
Please move the system usings to the top of the file.

Reply:
Fixed